### PR TITLE
Add Basic Chromatic Support

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/6487

### CODE REVIEW

**Comment by SudharakaP** [comment]
In the latest commit I've added Chromatic to build only on the `components`, `.storybook` and `stories` folders. So if like to see the build of Chromatic please use the previous commit; [028ba40](https://github.com/opencollective/opencollective-frontend/pull/8679/commits/028ba400b86db5c00d18b3800942f356c613fb66)

File: README.md [hunk 1]
```diff
@@ -117,3 +117,9 @@ To add new files or dependencies to this exported package, have a look at [scrip
 If you have any questions, ping us on Slack
 (https://slack.opencollective.com) or on Twitter
 ([@opencollect](https://twitter.com/opencollect)).
+
+## Thanks
+
+<a href="https://www.chromatic.com/"><img src="https://user-images.githubusercontent.com/321738/84662277-e3db4f80-af1b-11ea-88f5-91d67a5e59f6.png" width="153" height="30" alt="Chromatic" /></a>
+
+Thanks to [Chromatic](https://www.chromatic.com/) for providing the visual testing platform that helps us review UI changes and catch visual regressions.
```
Comment (Betree) [inline README.md:125]: I suppose that's a condition to be part of the OSS plan?

File: stories/host-dashboard/UnhostAccountModal.stories.mdx [hunk 2]
```diff
@@ -20,7 +23,11 @@ export const DefaultStory = props => <UnhostAccountModal {...props} />;
       },
     }}
   >
-    <UnhostAccountModal collective={webpackCollective} host={webpackCollective.host} />
+    <UserContext.Provider value={{ LoggedInUser: loggedInUser, loadingLoggedInUser: false }}>
+      <ToastProvider>
+        <UnhostAccountModal collective={webpackCollective} host={webpackCollective.host} />
+      </ToastProvider>
+    </UserContext.Provider>
```
Comment (Betree) [inline stories/host-dashboard/UnhostAccountModal.stories.mdx:30]: Ok for the `UserContextProvider` as it's difficult to figure a right default, but the `ToastProvider` should go in the [default decorators](https://github.com/opencollective/opencollective-frontend/blob/07f5fe7700d865f54f175fe6afccaf683d259631/.storybook/preview.js#L1) no?

**Review by Betree — APPROVED** [review]
Looking good, happy to give this a try

File: README.md (hunk 1 above)
Comment (SudharakaP) [reply README.md:125]: Yes, correct. 👍🏽

File: stories/host-dashboard/UnhostAccountModal.stories.mdx (hunk 2 above)
Comment (SudharakaP) [reply stories/host-dashboard/UnhostAccountModal.stories.mdx]: Oh, that makes sense. I've made the change. 👍🏽

**Pull request merged**